apps/web/src/components/workflow/FlowEditor.tsx

```diff
@@ -162,6 +162,8 @@ export function FlowEditor({
         const oldNode = nodes[i + 1] || { position: { x: 0, y: 120 } };
         const newId = (step._id || step.id) as string;
 
+        await triggerErrors('steps');
```

Comment:
Why this one is needed here? It triggers the form validation if i understand correctly?

Reply:
This is to get the meaningful errors before the submit of the form. So we would know what are the errors immediately on adding a node @scopsy